tide-log/Cargo.toml

```diff
@@ -18,7 +18,7 @@ repository = "https://github.com/rustasync/tide"
 version = "0.1.0"
 
  [dependencies]
-tide = { path = "../" }
+tide-core = { path = "../tide-core" }
```

Comment:
`tide-log` will need `tide` added to its `dev-dependencies` for the examples to compile (https://travis-ci.org/rustasync/tide/jobs/535258952#L872-L876)

Reply:
Added to all crates, since pretty much all of them are going to require it for documentation at some point anyway.